src/Components/OATPropertyEditor/Internal/PropertiesModelSummary.tsx

```diff
@@ -219,34 +240,38 @@ export const PropertiesModelSummary: React.FC<IPropertiesModelSummaryProps> = (
             setModelPath(value.trim());
         }
     }, []);
-    const onChangeRelationshipName = useCallback((_ev, value: string) => {
-        if (
-            selectedItem &&
-            isDTDLReference(selectedItem) &&
-            isValidReferenceName(value.trim(), selectedItem['@type'], false)
-        ) {
-            setRelationshipName(value.trim());
-        }
-    }, []);
-
-    // needed primarly for the version spinner since it behaves differently and you don't have to set focus
-    const forceUpdateId = useCallback(
-        ({ modelName, path, version }: IPartialModelId) => {
-            const newId = buildModelId({
-                modelName: modelName || modelUniqueName,
-                path: path || modelPath,
-                version: Number(version || modelVersion)
-            });
-            commitModelIdChange(newId);
+    const onChangeVersion = useCallback(
+        (_ev, value: string) => {
+            if (
+                isValidModelVersion(
+                    value.trim(),
+                    getDtdlVersionFromContext(modelContext),
+                    false
+                )
+            ) {
+                setModelVersion(value.trim());
+            }
+        },
+        [modelContext]
```

Comment:
checking dependency list if it is important, isValidModelVersion, getDtdlVersionFromContext, setModelVersion